logic/src/commonMain/kotlin/com/wire/kalium/logic/functional/Either.kt

```diff
@@ -29,14 +31,14 @@ import kotlin.contracts.contract
  * @see Left
  * @see Right
  */
+
```

Comment:
why this line removal?

Reply:
I left comment that I was reverting some changes in Either to discuss it on AC